sandbox-templates/mdx-loader-create-react-app/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mdx-loader-create-react-app",
+  "version": "1.0.0",
+  "private": true,
+  "main": "src/index.js",
+  "dependencies": {
+    "@mdx-js/loader": "2.0.0-rc.2",
+    "react": "17.0.2",
```

Comment:
Loose?

Reply:
I'm cautious, each sandbox would ideally have an exact version not a range.
So future people reviewing it will get an identical (or near identical) result.